src/sync/PostSyncProcessor.js

```diff
@@ -205,6 +205,12 @@ export class PostSyncProcessor {
       }
     }
 
+    if (record.type === 'customer_credit' && !record.isFinalised) {
+      funcs.push(() => {
+        record.finalise();
+      });
```

Comment:
Just a comment I reckon, it's a pretty odd thing ya know

Reply:
Yup, good point. 252aa6c.